packages/expo-image/ios/ImageModule.swift

```diff
@@ -130,6 +130,22 @@ public final class ImageModule: Module {
       }
     }
 
+    AsyncFunction("getBlurhashAsync") {
+      (url: URL, numberOfComponents: CGSize, promise: Promise) in
+      var downloader = SDWebImageDownloader()
+      downloader.downloadImage(with: url, progress: nil, completed: { image, _, _, _ in
+        DispatchQueue.global(qos: .userInitiated).async {
```

Comment:
Hmm, I'm not sure which QoS to use. I would choose the default one, but maybe the userInitiated is better. What do you think? cc @tsapeta

Reply:
Yeah, we should use the QoS that is appropriate to what we're doing. In many cases it's best to use the default (simply skip `qos` argument), which would inherit the QoS from the current queue.
But, I wonder, do we really need to dispatch it to a different queue? Does `downloadImage` run the callback on the main thread?

Reply:
Yes, it caused UI freezes (at least on Simulator) when generating blurhash without it.